tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -3230,25 +3251,74 @@ private void handleRelative(
+        String dateFormat = cldrFile.getWinningValue(availableFormatPath);
         SimpleDateFormat sdf = icuServiceBuilder.getDateFormat("gregorian", dateFormat);
         String sampleDate = sdf.format(DATE_SAMPLE);
-        String example1 =
-                value.substring(0, 1).toUpperCase() + value.substring(1) + " (" + sampleDate + ")";
-        String example2 = sampleDate + " (" + value + ")";
+
+        String contextTransformPath =
+                "//ldml/contextTransforms/contextTransformUsage[@type=\"relative\"]/contextTransform[@type=\"stand-alone\"]";
+        String contextTransformValue = cldrFile.getWinningValue(contextTransformPath);
+        String example1 = sampleDate + " (" + value + ")"; // value in middle-of-sentence usage
+        if (contextTransformValue != null && contextTransformValue.equals("titlecase-firstword")) {
+            value =
+                    TITLECASE.apply(
+                            Locale.forLanguageTag(getCldrFile().getLocaleID()),
+                            null,
+                            value); // locale-sensitive titlecasing
+        }
+        String example2 =
+                value
+                        + " ("
+                        + sampleDate
+                        + ")"; // value in stand-alone usage, titlecasing per contextTransform
+
+        examples.add("Set letter case for top example:");
         if (parts.contains("relativeTimePattern")) { // has placeholder
             String count = parts.getAttributeValue(-1, "count");
-            String exampleCount = COUNTS.get(count);
+            // Pick an appropriate example for this count, depends on the locale's plural rules
+            List<String> exampleCounts = COUNTS.get(count);
+            String exampleCount = exampleCounts.get(0); // default example for count
+            // if default example, does not work for the locale, override below
+            if (exampleCounts.size() > 1) {
+                int exampleCountSize = exampleCounts.size();
+                DecimalQuantitySamples samples =
+                        pluralInfo.getPluralRules().getDecimalSamples(count, SampleType.INTEGER);
+                if (samples == null) {
+                    // this locale has no integer samples for this count so use the decimal fraction
+                    // example at the end of the list
+                    exampleCount = exampleCounts.get(exampleCountSize - 1);
+                } else {
+                    Map<Count, String> countToStringExamplesMap =
+                            pluralInfo.getCountToStringExamplesMap();
+                    String stringExamples = countToStringExamplesMap.get(Count.valueOf(count));
+                    // skip the default value already set
+                    for (int i = 1; i < exampleCountSize; i++) {
+                        String exampleCountTest = exampleCounts.get(i);
+                        if (stringExamples.contains(exampleCountTest)) {
```

Comment:
This is a little hokey (searching for a string value in the string of sample values) but it works with the values used here. Could also convert the string value to a number and get the count from PluralRules and see whether that matches.